core/domain/story_validators.py

```diff
@@ -196,7 +196,7 @@ def _validate_node_titles(
                         item.id, model_id, model_class.__name__, model_id))
                 continue
             nodes = story_model.story_contents['nodes']
-            node_titles = [node.title for node in nodes]
+            node_titles = [node['title'] for node in nodes]
```

Comment:
Can you please confirm that we have a test for this code? If yes, then how we missed this issue?

Reply:
I have tested and confirmed the occurrence of the error without the change and the correct working after the change. It is quite weird how this issue was missed.

Reply:
@rohitkatlaa Did you added a test that would catch this error?

Reply:
I have added a test. Done.